### ISSUE TITLE
Dockerize the app 

### ISSUE THREAD

**Issue by vic778** [BenMukebo/health-care-online-system#14]
I've seen that we have some ruby version incompatibility and PostgreSQL setups different from each other. I think we can use docker for a middle setup for all of us and a safe work environment to avoid the `ruby version`, `pg setup`, and for the stronger env in the prod mode.
- setup docker 
- containerize the image 
- build the image

**Comment by BenMukebo** [comment]
@vic778, You're right, I'm going to finish my Dockar courses then you will give me a review!!
- This will be my next task, an opportinity for me to learn dockar 👍🏻

**Comment by vic778** [comment]
> @vic778, You're right, I'm going to finish my Dockar courses then you will give me a review!!
> 
> * This will be my next task, an opportinity for me to learn dockar 👍🏻

Let's enjoy it then
